docker/templates/Dockerfile.j2

```
@@ -16,13 +16,15 @@
 {% if image_flavor == 'ubi8' -%}
   {% set base_image = 'docker.elastic.co/ubi8/ubi-minimal' -%}
   {% set package_manager = 'microdnf'  -%}
+  {% arch_command = 'uname -m' -%}
```

I think we're missing a set here?

```suggestion
  {% set arch_command = 'uname -m' -%}
```